lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid;
+  border-radius: $nx-border-radius;
+  display: flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  max-width: 860px;
```

Comment:
I'm wondering whether there's really value in making this explicit.  It was reverse engineered from what we think the max width of a valid policy name could be, so it might be better to just leave it up to that.

Reply:
> I'm wondering whether there's really value in making this explicit. It was reverse engineered from what we think the max width of a valid policy name could be, so it might be better to just leave it up to that.

I only had it there bc it was explicitly mentioned in the Zeplin file, I'd prefer that max-widths be handled by the container wherever possible and that should be the case here and so I have removed it: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/77791f37d04ce6a2ab9d195907412b2c1fadb211